.circleci/config.yml

```diff
@@ -139,9 +139,30 @@ workflows:
       - upload-coverage:
           requires:
             - build
+  build-test-deploy:
+    jobs:
+      - setup-coverage
+      - build:
+          requires:
+            - setup-coverage
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+/
+      - upload-coverage:
+          requires:
+            - build
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+/
       - deploy:
           requires:
             - upload-coverage
           filters:
             branches:
-              only: master
+              ignore: /.*/
+            tags:
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+/
```

Comment:
Do we just want to have tags in the deploy job? Could you clarify why it's in the other jobs?

Reply:
Thanks for explaining!